### PR TITLE
docs: Update documentation content and toc structure for Toolkit

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## Description

Update documentation content and toc structure for Toolkit

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Uno Toolkit
-A set of custom controls for the UWP, WinUI and the Uno Platform not offered out of the box by WinUI, such as Card, TabBar, NavigationBar, etc.
+A set of custom controls for the UWP, WinUI, and the Uno Platform not offered out of the box by WinUI, such as Card, TabBar, NavigationBar, etc.
```
Comment (kazo0) [inline README.md:2]: I'm wondering if we should start to phase out the use of UWP and just stick with WinUI going forward?

File: doc/getting-started.md [hunk 2]
```diff
@@ -35,7 +38,7 @@ The Uno.Toolkit.Material library is available as NuGet packages that can be adde
 		</Application.Resources>
 	</Application>
 	```
-	> The styles for the new controls are found in both ToolkitResources (design-agnostic variants) and MaterialToolkitResources(Material Design variants).
+	> The styles for the new controls are found in both ToolkitResources (design-agnostic variants) and MaterialToolkitResources (Material Design variants).
```
Comment (jeromelaban) [inline doc/getting-started.md:41]:
```suggestion
	> [!TIP]
	> The styles for the new controls are found in both ToolkitResources (design-agnostic variants) and MaterialToolkitResources (Material Design variants).
```

File: README.md (hunk 1 above)
Comment (jeromelaban) [reply README.md:2]:
```suggestion
Uno Toolkit provides a set of higher-level UI Controls designed specifically for multi-platform, responsive applications.
```

Comment (agneszitte-nventive) [reply README.md]: Thanks @jeromelaban !

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**